runtime/doc/lsp.txt

```diff
@@ -904,7 +904,7 @@ Lua module: vim.lsp.client                                        *lsp-client*
       • {rpc}                   (`vim.lsp.rpc.PublicClient`) RPC client
                                 object, for low level interaction with the
                                 client. See |vim.lsp.rpc.start()|.
-      • {offset_encoding}       (`string`) The encoding used for communicating
+      • {position_encoding}     (`string`) The encoding used for communicating
```

Comment:
Since this is breaking, we don't need to make this change. Instead, as a bridge, let's just mention "position encoding" in the docs so that it can be found by searching, and to help users understand that `offset_encoding` is an alias to the spec's "position encoding".
```suggestion
      • {offset_encoding}     (`string`) Called "position encoding" in LSP spec. The encoding used for communicating
```